tests/settings.py

```diff
@@ -25,4 +25,22 @@
 
 DEBUG = True
 
-STATIC_URL = "/static/"
\ No newline at end of file
+STATIC_URL = "/static/"
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
```

Comment:
Come to think of it, this shouldn't really be in this PR at all. My previously misconfigured tox setup installed 1.10a1, which failed miserably.